src/services/api.tsx

```diff
@@ -26,4 +26,14 @@ export const updateRoomsUrl = `${baseUrl}/admin/rooms/` //with id
 export const deleteRoomsUrl = `${baseUrl}/admin/rooms/` //with id
 // ******* Get All Rooms Facility*********
 export const facilitiesRoomsUrl = `${baseUrl}/admin/room-facilities`;
```

Comment:
You need to change the name of this url in the facilities page.

Reply:
already changed in facilities by Samiya but still not merge

Reply:
> already changed in facilities by Samiya but still not merge

The application should function properly without any errors in the branch before merging, but currently, if anyone moved to the branch , there will be multiple minor errors like this :
![image](https://github.com/SAMIYAghb/hotel-system/assets/64327685/4faf5ae3-007f-46e7-a3c9-5a051e1e0fcb)
![image](https://github.com/SAMIYAghb/hotel-system/assets/64327685/c55db8de-b251-4a41-bfb6-3ff4a804d4a4)

and this indicates that the code in your local machine is not the same as the code in the branch.